### DISCUSSION TITLE
a question about same-different-image plugin

### DISCUSSION THREAD

**Discussion by mecui22** [jspsych/jsPsych#1552] [Q&A]
Hi! Sorry if someone has already asked this question!

For my study, in each trial, I have to have this sequence: 1) image 1, 2) image 2, and 3) prompt to response. However, the built-in prompt from the same different image plugin appears with the second image simultaneously. 

Any suggestion? I hope to keep using this plugin since I need to pair up images conditionally. 

Thank you! Cheers,

**Comment by jodeleeuw** [comment]
Hi @mecui22,

This might be a case where you are better served by using a timeline with [timeline variables](https://www.jspsych.org/overview/timeline/#timeline-variables). The docs describe a similar kind of situation, where you need to pair two separate stimuli together. 

In your case it sounds like three separate trials might work. The first two display the two images, the last to get a response.
